DOCS-5584 brand intro (#96)
mulesoft/docs-api-community-manager: modules/ROOT/pages/configure-acm.adoc

=== Enable Chatter Tracking for APIs [[enable-chatter-tracking]]

> Chatter tracking enables creating discussions on specific objects like APIs. To use Chatter tracking for APIs:

Reviewer: ```suggestion
Chatter tracking enables your community to create discussions on specific objects like APIs. To use Chatter tracking for APIs:
```
enable people

==== Add Self Register Component to Registration Page [[add-self-register-custom-component-to-register-page]]

> The self-register component enables guest users to self-register in your community to become members. If you're not using any of the ACM out-of-the-box templates, follow these steps to use the ACM-specific self-registration component.

Reviewer: ```suggestion
The self-register component enables guest users to register themselves as members of your community. If you're not using any of the ACM out-of-the-box templates, follow these steps to use the ACM-specific self-registration component:
```
